electron-app/src/components/dashboard/ManualBalanceCardList.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <v-list-item
+    :id="`${name}_box`"
+    :ripple="false"
+    class="manual-balance-box__item"
+    to="/blockchain-accounts"
+    @click="doNothing"
+  >
+    <v-list-item-avatar tile class="manual-balance-box__icon">
+      <v-icon>
+        fa fa-{{ manualBalanceIcons[name.replace(/\s+/g, '')] }}
+      </v-icon>
+    </v-list-item-avatar>
+    <v-list-item-content>
+      <v-list-item-title class="d-flex justify-space-between">
+        <span>
+          {{ name | capitalize }}
+        </span>
+        <span class="text-end">
+          <amount-display
+            show-currency="symbol"
+            :fiat-currency="currency.ticker_symbol"
+            :value="amount"
+          ></amount-display>
+        </span>
+      </v-list-item-title>
+    </v-list-item-content>
+  </v-list-item>
+</template>
+
+<script lang="ts">
+import { default as BigNumber } from 'bignumber.js';
+import { Component, Prop, Vue } from 'vue-property-decorator';
+import { createNamespacedHelpers } from 'vuex';
+import AmountDisplay from '@/components/display/AmountDisplay.vue';
+import { Currency } from '@/model/currency';
+
+const { mapGetters } = createNamespacedHelpers('session');
+
+@Component({
+  components: { AmountDisplay },
+  computed: {
+    ...mapGetters(['currency'])
+  }
+})
+export default class ManualBalanceCardList extends Vue {
+  @Prop({ required: true })
+  name!: string;
+  @Prop({ required: true })
+  amount!: BigNumber;
+
+  currency!: Currency;
+  manualBalanceIcons = {
+    external: 'book',
+    banks: 'bank',
+    equities: 'suitcase',
+    realestate: 'home',
+    commodities: 'shopping-basket',
+    blockchain: 'link'
+  };
+
+  doNothing() {}
+}
+</script>
+<style scoped lang="scss">
+.manual-balance-box__currency__symbol {
```

Comment:
while some of the old components are not yet there, could we do the nesting for the new components?